code/datums/status_effects/debuffs.dm

```diff
@@ -200,6 +200,8 @@
 	. = ..()
 	if(usr != owner)
 		return
+	if(owner.stat == DEAD)
```

Comment:
Just check either owner.stat or owner.incapacited, you shouldnt be able to do it while unconscious either